spec/features/station/user_can_create_station_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../spec_helper'
+
+RSpec.describe "when user visits new station path" do
+  it "user can create new station" do
+
+    visit('/stations/new')
+
+    fill_in 'station[name]', with: "Station1"
+    fill_in 'station[dock_count]', with: 2
+    fill_in 'station[city]', with: "Auckland"
+    fill_in 'station[installation_date]', with: "2015-09-06"
+
+    click_on 'Create New Station'
+
+    station = Station.last
+    expect(station.name).to eq("Station1")
+    expect(page).to have_content("Auckland")
+    expect(current_path).to eq("/stations/#{station.id}")
```

Comment:
I would consider "scoping" the expectations to a particular css selector with the `within` block that capybara provides. Something like:

```ruby
withing(".station_information") do
  expect(page).to have_content("Station1")
  expect(page).to have_content("Auckland")
end
```

Also I would move the current_path expectation up right after the `click_on 'Create new Station'` method. This change will help because if you fail for being on the wrong page the error is more helpful that if you fail for not having the expected content.